src/caselawclient/factories.py

```diff
@@ -70,13 +71,18 @@ def build(
         api_client: Optional[MarklogicApiClient] = None,
         **kwargs: Any,
     ) -> target_class:
+        def _fake_linked_documents(*args: Any, **kwargs: Any) -> list["Document"]:
+            return [document]
+
         if not api_client:
             api_client = Mock(spec=MarklogicApiClient)
             api_client.get_judgment_xml_bytestring.return_value = DEFAULT_DOCUMENT_BODY_XML.encode(encoding="utf-8")
             api_client.get_property_as_node.return_value = None
 
         document = cls.target_class(uri, api_client=api_client)
         document.body = kwargs.pop("body") if "body" in kwargs else DocumentBodyFactory.build()
+        document.identifiers.add(FindCaseLawIdentifier(value="a1b2c3"))
```

Comment:
This should really behave similarly to `body`, so maybe something like

```suggestion
if "identifiers" in kwargs:
    for identifier in kwargs.pop("identifiers"):
        document.identifiers.add(identifier)
else:
   
 document.identifiers.add(FindCaseLawIdentifier(value="a1b2c3"))
```

and make sure `identifier` is an `Optional[list[Identifier]]`?